src/quo2/components/counter/counter.cljs

```diff
@@ -38,7 +38,8 @@
                      1     16
                      2     20
                      28)]
-    [rn/view {:accessible true
+    [rn/view {:test-ID "counter-component"
```

Comment:
I tried changing `:test-ID` to a keyword and it works just fine in production code. Could you please change all current test IDs to keywords? This is more idiomatic in Clojure. Not in the tests of course, because that layer doesn't like certain Clojure idioms.

Reply:
cool, yeah I will try it out! I'm pretty sure it should work